EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionsBrowserViewModel.cs

```diff
@@ -588,8 +647,8 @@ private void UpdateElementDefinition()
                 topElementDefinitionNew.IsTopElement = true;
 
                 // clear old top element.
-                if (topElementDefinitionOld != null &&
-                    topElementDefinitionOld != topElementDefinitionNew)
+                if ((topElementDefinitionOld != null) &&
```

Comment:
The new brackets are redundant no?

Reply:
That is recommendation from resharper as a clarity rule when two or more conditions are present inside if.